pkg/scheduler/core/extender_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/defaultbinder"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/queuesort"
+	schedulerruntime "k8s.io/kubernetes/pkg/scheduler/framework/runtime"
```

Comment:
@ahg-g Appreciate for the feedback.

I too would have liked to use runtime as the name, but resorted to `schedulerruntime` because some files import the package 

`runtime "k8s.io/apimachinery/pkg/runtime"` or
"k8s.io/apimachinery/pkg/runtime"

which would conflict with the name. Example in `pkg/scheduler/factory.go`.

I look forward to hearing your suggestion on what to do next. apologies for the delay, will look forward to resolving this PR asap.

Reply:
I guess we have three options:
1) use ```frameworkruntime```, long but descriptive.
2) use a shorter name ```fwkruntime```
3) use ```runtime``` for ```framework/runtime``` and rename pkg ```apimachinery/pkg/runtime``` to something else like ```machineryruntime``` or ```apiruntime```

@Huang-Wei @alculquicondor what do you think? any other suggestions? perhaps option 2 is the least evil?

Reply:
I went with `fwkruntime` although when I think about it the alias `schedulerframework` is already used which is even longer.

Reply:
sg